common/test/acceptance/tests/lms/test_lms_courseware.py

```diff
@@ -441,7 +440,7 @@ class CoursewareMultipleVerticalsTest(CoursewareMultipleVerticalsTestBase):
     Test courseware with multiple verticals
     """
 
-    @skip('Disable temporarily to get course bookmarks out')
+    @flaky  # PLAT-1198; should be fixed, but verify that failures stop before removing
```

Comment:
Note that you can run the test 10 times in Jenkins by using the following notation:

 @flaky(max_runs=10, min_passes=10):

You can up that to 20 or 30 and run just the specific test using
https://build.testeng.edx.org/view/edx-platform-specific-tests/

That way you can verify the test passes 100+ times before you submit the PR (and remove the flaky decorator with the submission).